apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-common-spring5/src/main/java/co/elastic/apm/agent/webfluxcommon/SpringWeb5Utils.java

```diff
@@ -29,8 +30,13 @@
 public class SpringWeb5Utils implements SpringWebVersionUtils.ISpringWebVersionUtils {
 
     @Override
-    public int getStatusCode(Object response) {
+    public int getServerStatusCode(Object response) {
         HttpStatus statusCode = ((ServerHttpResponse) response).getStatusCode();
         return statusCode != null ? statusCode.value() : 200;
     }
+
+    @Override
+    public int getClientStatusCode(Object response) {
+        return ((ClientResponse) response).rawStatusCode();
```

Comment:
rawStatusCode() has been removed in favor of .statusCode().value() if I recall correctly